[2025-02-12T22:36:28.174+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T22:36:28.188+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T13:11:00+00:00 [queued]>
[2025-02-12T22:36:28.195+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T13:11:00+00:00 [queued]>
[2025-02-12T22:36:28.195+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T22:36:28.222+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-04 13:11:00+00:00
[2025-02-12T22:36:28.227+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=16098) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T22:36:28.228+0000] {standard_task_runner.py:72} INFO - Started process 16107 to run task
[2025-02-12T22:36:28.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-04T13:11:00+00:00', '--job-id', '5661', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpfkoosrfs']
[2025-02-12T22:36:28.228+0000] {standard_task_runner.py:105} INFO - Job 5661: Subtask fetch_and_store_crypto_data
[2025-02-12T22:36:28.262+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T13:11:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T22:36:28.340+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-04T13:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-04T13:11:00+00:00'
[2025-02-12T22:36:28.340+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T22:36:28.686+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 8, 'minute': 55, 'hour': 4245, 'day': 5139, 'month': 5139, 'total_calls': 5139}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T22:36:28.687+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T22:36:28.698+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T22:36:28.699+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T22:36:28.705+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T22:36:28.705+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-04T13:11:00+00:00, execution_date=20231004T131100, start_date=20250212T223628, end_date=20250212T223628
[2025-02-12T22:36:28.762+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T22:36:28.773+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T22:36:28.775+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
